configs/otelcol-contrib.yaml

```diff
@@ -0,0 +1,55 @@
+extensions:
```

Comment:
This is a copy from the core's config. We do not currently ship contrib with a default config file, and I think this is a good opportunity to fix that.